src/main/java/com/puppycrawl/tools/checkstyle/filters/JavaPatchFilterElement.java

```diff
@@ -53,6 +53,10 @@
     static {
         CHECK_TO_ANCESTOR_NODES_MAP.put("AvoidNestedBlocks",
                 Arrays.asList(TokenTypes.SLIST));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("DefaultComesLast",
+                Arrays.asList(TokenTypes.METHOD_DEF));
+        CHECK_TO_ANCESTOR_NODES_MAP.put("DeclarationOrder",
+                Arrays.asList(TokenTypes.CLASS_DEF));
```

Comment:
Going up to CLASS_DEF is almost neverSuppressedCheck. But I don't know how we can handle it otherwise :)
Please add `INTERFACE_DEF` and `ENUM_DEF` as well.

Reply:
ok, done